apps/hash-graph/.justfile

```diff
@@ -31,7 +31,40 @@ deployment-up *arguments: docker-build
 deployment-down *arguments:
   @just yarn external-services down {{arguments}}
 
+[private]
+test *arguments:
+  @just deployment-up
+  @just --justfile {{repo}}/.justfile test {{arguments}}
+  cargo test -p graph-benches --benches --profile {{profile}} {{arguments}}
+  @just deployment-up graph --wait
+  @just yarn httpyac send --all {{repo}}/apps/hash-graph/tests/rest-test.http
+  @just deployment-down
+  @just generate-openapi-client
+
+[private]
+coverage *arguments:
+  @just deployment-up
+  cargo llvm-cov --workspace --all-features --all-targets {{arguments}}
+  @just deployment-down
+
+[private]
+bench *arguments:
+  @just deployment-up
+  @just --justfile {{repo}}/.justfile bench {{arguments}}
+  @just deployment-down
+
+[private]
+miri *arguments:
+  @echo 'miri is disabled for `hash-graph`'
+
 # Generates the OpenAPI client for the Graph REST API
+[private]
+generate-openapi-client-shared:
+  just yarn workspace @local/hash-graph-client-generator generate
+  just yarn workspace @local/hash-graph-client prettier --write .
+  just yarn workspace @local/hash-graph-client fix:eslint
```

Comment:
Sorry, that I have not noticed that before...
```suggestion
  @just yarn workspace @local/hash-graph-client-generator generate
  @just yarn workspace @local/hash-graph-client prettier --write .
  @just yarn workspace @local/hash-graph-client fix:eslint
```

Reply:
Oops now that we removed the shebang, good point

Reply:
[Quiet generate-openapi-client-shared steps](https://github.com/hashintel/hash/pull/2374/commits/4084a8097aef3c840bf1223c5b845be906e2d49f)